lib/tizen_plugins.dart

```diff
@@ -253,17 +264,16 @@ const List<String> _kKnownPlugins = <String>[
 ///
 /// See: [FlutterProject.ensureReadyForPlatformSpecificTooling] in `project.dart`
 Future<void> ensureReadyForTizenTooling(FlutterProject project) async {
-  if (!project.directory.existsSync() ||
-      project.hasExampleApp ||
-      project.isPlugin) {
-    return;
-  }
```

Comment:
I wanted to the `ensureReadyForPlatformSpecificTooling()` method to be called even in the case of the plugin project to generate `XXXX.csproj.user` file.

Reply:
Is `ensureReadyForTizenTooling` always called on plugin projects? I can't find where it is invoked.

Reply:
`ensureReadyForTizenTooling` is called always in case of `pub get` command.

https://github.com/flutter-tizen/flutter-tizen/blob/a310784768a6b6ecbb844a3fc70177bb4a6df262/lib/tizen_pub.dart#L118

Reply:
I know that but is it called on plugin projects?

Reply:
Yes, originally if the project type is a plugin, just skipped by the remove lines. But I want to call `ensureReadyForPlatformSpecificTooling()` method to generate `XXXX.csproj.user` file.

Reply:
I reverted the changes of this method and add new `_ensurePluginsReadyForTizenTooling` method to generate `.csproj.user`  files of dependent dotnet plugins.

Reply:
> ~~I reverted the changes of this method and add new `_ensurePluginsReadyForTizenTooling` method to generate `.csproj.user` files of dependent dotnet plugins.~~

`ensureReadyForPlatformSpecificTooling()` should be called when a plugin project is created and `_ensurePluginsReadyForTizenTooling()` method will be called for depedent dotnet plugins.

Reply:
Can we revert this change (calling `ensureReadyForPlatformSpecificTooling` on a plugin project) to avoid ambiguity? In either way `_ensurePluginsReadyForTizenTooling` will be called on dependent dotnet plugins after `flutter-tizen create -t plugin`, because `ensureReadyForTizenTooling` will be called on the example project of the created plugin.

Reply:
I wanted the plugin project to be able to build even before modifying the `pubspec.yaml` file. If this change is reverted, the `.csproj.user` file will be created only when the correct plugin is defined in pubspec.yaml after creating the project. If this doesn't matter, it's possible to turn it back.

Reply:
I think it's better to be explicit so I prefer calling `ensureReadyForPlatformSpecificTooling` on app projects only.

Reply:
Ok, I agree with you. How about moving the code for generating `.csproj.user` file to `tizen_pub.dart`?
I hope that the plugin project can be built immediately after the project is created.
for example:
```dart
  Future<void> _postPub(String directory) async {
    final FlutterProject project = directory == null
        ? FlutterProject.current()
        : FlutterProject.fromDirectory(_fileSystem.directory(directory));
    await ensureReadyForTizenTooling(project);

    final TizenProject tizenProject = TizenProject.fromFlutter(project);
    if (project.isPlugin &&
        tizenProject.existsSync() &&
        tizenProject.isDotnet) {
      updateDotnetUserProjectFile(tizenProject.projectFile, skipIfExists: true);
    }
  }
```

Reply:
Please ignore my last comment.. 
~~How about moving the code for generating .csproj.user file to tizen_pub.dart?~~
I will try to update the `.csproj.user` file in `create.dart` after creating the project.